src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+ 
   <body>
-    <h1>Search bar airbnb</h1>
+        
+    <form  action="https://mate-academy-form-lesson.herokuapp.com/create-application"  
+    method="post"  
+    data-qa="big"
+    class="form"
+    >  
+      <img 
+        src="images/Search.svg" 
+        alt="image"
+        class="searchimage"
+      >
+      <input 
+        type="text"
+        class="field field_focus"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
```

Comment:
use classes that will show an element connection (according to BEM-methodoogy)
```suggestion
      <img 
        src="images/Search.svg" 
        alt="image"
        class="form__image"
      >
      <input 
        type="text"
        class="form__input"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```

Reply:
Not resolved